README.md

```diff
@@ -494,12 +494,13 @@ Shows all the versions of PostgreSQL available to download and build. Handy to
 help you finding a version to pass to the `build` command. Note that the
 `available` command produces copious output.
 
-```
-$ pgenv available
-...
+``` sh
```

Comment:
Not really shell, because none of the output is shell.

Reply:
The same for the very first piece of code that reports `git`. I think, since all examples and codes are shell-related, we should apply `sh` everywhere (or either remove it from everywhere).

Reply:
Fix in beb5f98